MANUAL.md

```diff
@@ -1,12 +1,35 @@
 # Welcome to the Circuit user manual
```

Comment:
@juanecabellob Could you link to this headline directly? like this: [MANUAL.md#welcome-to-the-circuit-user-manual](#welcome-to-the-circuit-user-manual)